<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">CreditWindow {
 background-color: rgb(207, 248, 255);
}
</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background-color: rgb(255, 157, 246);
 color: white;
 border: 1px solid gray;
font: 18pt &quot;Rockwell&quot;;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
}
</string>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_cr_amount_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>          Total credit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="term_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                      Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                      Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthly_payment_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>            Monthly payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>       Overpayment on credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_payment_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>               Total payment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background-color: rgb(227, 167, 255);
 color: white;
 border: 1px solid gray;
font: 18pt &quot;Rockwell&quot;;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="type_groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(189, 118, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="annuity_radio_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Rockwell&quot;;
 background-color: rgb(189, 118, 255);</string>
    </property>
    <property name="text">
     <string>        annuity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="differentiated_radio_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Rockwell&quot;;
 background-color: rgb(189, 118, 255);</string>
    </property>
    <property name="text">
     <string>   differentiated</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="total_cr_amount_lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="term_lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_rate_lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthly_payment_lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overpayment_lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_payment_lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>510</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background-color: rgb(175, 174, 255);
 color: white;
 border: 1px solid gray;
 font: 18pt &quot;Rockwell&quot;;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="months_lbl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>months</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent_lbl">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
